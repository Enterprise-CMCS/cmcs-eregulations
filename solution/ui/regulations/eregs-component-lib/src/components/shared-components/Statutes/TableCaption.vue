<template>
    <div class="table__caption">
        This table shows U.S. Code sections enacted by the Social Security Act.
        See also:
        <a
            target="_blank"
            rel="noopener noreferrer"
            class="external"
            href="https://uscode.house.gov/view.xhtml?req=granuleid%3AUSC-prelim-title42-chapter7&amp;saved=%7CZ3JhbnVsZWlkOlVTQy1wcmVsaW0tdGl0bGU0Mi1jaGFwdGVyNy1mcm9udA%3D%3D%7C%7C%7C0%7Cfalse%7Cprelim&amp;edition=prelim"
        >full table of contents and text for 42 U.S.C. Chapter 7 (Social
            Security)</a>.<br>
        Learn about these sources:
        <a
            href="https://uscode.house.gov/"
            target="_blank"
            rel="noopener noreferrer"
            class="external"
        >US Code House.gov</a>,
        <a
            href="https://www.govinfo.gov/app/collection/comps/"
            target="_blank"
            rel="noopener noreferrer"
            class="external"
        >Statute Compilation</a>,
        <a
            href="https://www.govinfo.gov/app/collection/uscode"
            target="_blank"
            rel="noopener noreferrer"
            class="external"
        >US Code Annual</a>,
        <a
            href="https://www.ssa.gov/OP_Home/ssact/ssact.htm"
            target="_blank"
            rel="noopener noreferrer"
            class="external"
        >SSA.gov Compilation</a>.
    </div>
</template>
