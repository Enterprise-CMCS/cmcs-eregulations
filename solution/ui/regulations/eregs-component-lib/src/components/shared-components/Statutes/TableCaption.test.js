import { render } from "@testing-library/vue";
import { describe, it, expect } from "vitest";

import TableCaption from "./TableCaption.vue";

describe("Statute Table Caption", () => {
    it(`Creates a snapshot of the Statute Selector with default props`, async () => {
        const wrapper = render(TableCaption);
        expect(wrapper).toMatchSnapshot();
    });
});
