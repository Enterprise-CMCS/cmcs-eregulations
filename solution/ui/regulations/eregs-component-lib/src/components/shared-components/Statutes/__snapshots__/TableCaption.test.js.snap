// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Statute Table Caption > Creates a snapshot of the Statute Selector with default props 1`] = `
{
  "baseElement": <body>
    <div>
      <div
        class="table__caption"
      >
         This table shows U.S. Code sections enacted by the Social Security Act. See also: 
        <a
          class="external"
          href="https://uscode.house.gov/view.xhtml?req=granuleid%3AUSC-prelim-title42-chapter7&saved=%7CZ3JhbnVsZWlkOlVTQy1wcmVsaW0tdGl0bGU0Mi1jaGFwdGVyNy1mcm9udA%3D%3D%7C%7C%7C0%7Cfalse%7Cprelim&edition=prelim"
          rel="noopener noreferrer"
          target="_blank"
        >
          full table of contents and text for 42 U.S.C. Chapter 7 (Social Security)
        </a>
        .
        <br />
         Learn about these sources: 
        <a
          class="external"
          href="https://uscode.house.gov/"
          rel="noopener noreferrer"
          target="_blank"
        >
          US Code House.gov
        </a>
        , 
        <a
          class="external"
          href="https://www.govinfo.gov/app/collection/comps/"
          rel="noopener noreferrer"
          target="_blank"
        >
          Statute Compilation
        </a>
        , 
        <a
          class="external"
          href="https://www.govinfo.gov/app/collection/uscode"
          rel="noopener noreferrer"
          target="_blank"
        >
          US Code Annual
        </a>
        , 
        <a
          class="external"
          href="https://www.ssa.gov/OP_Home/ssact/ssact.htm"
          rel="noopener noreferrer"
          target="_blank"
        >
          SSA.gov Compilation
        </a>
        . 
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="table__caption"
    >
       This table shows U.S. Code sections enacted by the Social Security Act. See also: 
      <a
        class="external"
        href="https://uscode.house.gov/view.xhtml?req=granuleid%3AUSC-prelim-title42-chapter7&saved=%7CZ3JhbnVsZWlkOlVTQy1wcmVsaW0tdGl0bGU0Mi1jaGFwdGVyNy1mcm9udA%3D%3D%7C%7C%7C0%7Cfalse%7Cprelim&edition=prelim"
        rel="noopener noreferrer"
        target="_blank"
      >
        full table of contents and text for 42 U.S.C. Chapter 7 (Social Security)
      </a>
      .
      <br />
       Learn about these sources: 
      <a
        class="external"
        href="https://uscode.house.gov/"
        rel="noopener noreferrer"
        target="_blank"
      >
        US Code House.gov
      </a>
      , 
      <a
        class="external"
        href="https://www.govinfo.gov/app/collection/comps/"
        rel="noopener noreferrer"
        target="_blank"
      >
        Statute Compilation
      </a>
      , 
      <a
        class="external"
        href="https://www.govinfo.gov/app/collection/uscode"
        rel="noopener noreferrer"
        target="_blank"
      >
        US Code Annual
      </a>
      , 
      <a
        class="external"
        href="https://www.ssa.gov/OP_Home/ssact/ssact.htm"
        rel="noopener noreferrer"
        target="_blank"
      >
        SSA.gov Compilation
      </a>
      . 
    </div>
  </div>,
  "debug": [Function],
  "emitted": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "html": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
